packages/snap/src/helpers/file-helpers.ts

```diff
@@ -21,10 +21,26 @@ export const getSnapFileBytes = async (path: string) => {
   return bytes;
 };
 
-export const getFileBytes = async (path: string) => {
-  try {
-    return await readBytesFile(new URL(path).href);
-  } catch (error) {
-    return await getSnapFileBytes(path);
+export const concatAndGetShardedFiles = async (
+  paths: string[],
+): Promise<Uint8Array> => {
+  const files = await Promise.all(paths.map(getSnapFileBytes));
+
+  return new Uint8Array(
+    files.reduce((acc, fileBytes) => {
+      return [...acc, ...fileBytes];
+    }, []),
+  );
+};
+
+export const getFileBytes = async (path: string | string[]) => {
+  if (typeof path === 'string') {
+    try {
+      return await readBytesFile(new URL(path).href);
+    } catch (error) {
+      return await getSnapFileBytes(path);
```

Comment:
fetch vs snap_getFile

Reply:
Why do we use `snap_getFile` as a fallback and not the other way around? Also, it's trivial to distinguish a URL from a local path instead of doing this

Reply:
You can just check the prefix of the path instead of checking for a failed request. This is not user input so it will be robust enough